app/models/carrier_profile.rb

```diff
@@ -6,4 +6,5 @@ class CarrierProfile
 
   field :fein, type: String
   field :profile_name, type: String
+  field :requires_employer_updates_on_enrollments, type: Boolean, default: false
```

Comment:
This flag should go on the "Carrier" model itself to make your life easier.  I'm 99% sure I probably gave you the wrong class name in the ticket.  It will be on the same level then as the other flags we are adding for this.

Reply:
Done updating. Moved field: requires_employer_updates_on_enrollments to carrier model and updated script that update field value.